src/transformers/modeling_outputs.py

```diff
@@ -1263,3 +1263,19 @@ class XVectorOutput(ModelOutput):
     embeddings: torch.FloatTensor = None
     hidden_states: Optional[Tuple[torch.FloatTensor]] = None
     attentions: Optional[Tuple[torch.FloatTensor]] = None
+
+
+@dataclass
+class BackboneOutput(ModelOutput):
+    """
+    Base class for outputs of backbones.
+
+    Args:
+        stage_names (`tuple(str)`):
+            Names of the stages.
+        hidden_states (`tuple(torch.FloatTensor)` of shape `(batch_size, num_channels, height, width)`):
+            Hidden states (also called feature maps) of the stages.
```

Comment:
Would it make more sense to call these `feature_maps` (we are creating a new API so it's the only chance we get to fix names if we want to)? I think it would be more natural to vision users no?

Reply:
Makes sense, will update!